tools/MultiProjectValidator/project.json

```diff
@@ -5,12 +5,17 @@
     "emitEntryPoint": true
   },
   "dependencies": {
-    "NETStandard.Library": "1.5.0-rc2-24027",
-    "Microsoft.DotNet.ProjectModel": { "target": "project" },
-    "Microsoft.DotNet.Cli.Utils": { "target": "project" }
+    "NETStandard.Library": "1.5.0-rc3-24113-00",
```

Comment:
While you are in here, could you change this to Microsoft.NETCore.App? It doesn't matter if it is the current version or N-1